tests/test_shortcuts.py

```diff
@@ -199,6 +199,12 @@ def test_run_show_cmd_logfile_stdout(self, mock_out):
         self.assertEqual(mock_out.getvalue(),
                          'COMMAND: echo foo\n-----------------\nfoo\n')
 
+    def test_run_split_in_middle_of_utf8_sequence(self):
```

Comment:
If you're only planning to add the test now and not a fix, would you like to add an xfail marker here?  Otherwise we can't merge it without breaking CI.

Reply:
Please don't merge this. I wanted provide a reproducer. I may get to writing a fix for this eventually, but not sure when.

Reply:
I added the xfail. If you want to, feel free to merge.